elyra/cli/pipeline_app.py

```diff
@@ -301,7 +304,7 @@ def run(pipeline_path):
     _validate_pipeline_file_extension(pipeline_path)
 
     pipeline_definition = \
-        _preprocess_pipeline(pipeline_path, runtime='local', runtime_config='local')
+        _preprocess_pipeline(pipeline_path, runtime='local')
```

Comment:
Why remove the explicit `runtime_config` here? it sort of explicitly say that run is only happening locally.